app/src/androidTest/java/org/mozilla/focus/activity/DefaultBrowserTest.java

```diff
@@ -57,6 +57,16 @@ public void setup() {
         uiDevice = UiDevice.getInstance(getInstrumentation());
     }
 
+    /**
+     * Test case no: TC0090
+     * Test case name: Set FIrefox Lite as default browser
+     * Steps:
+     * 1. Set Chrome as a default browser
+     * 2. Launch Rocket
+     * 3. Go to settings page (Tap menu -> settings)
+     * 4. Tap "Make default browser"
+     * 5. Go back to Rocket's settings
+     * 6. Check it correctly set default browser to Firefox Lite */
```

Comment:
the closing */ should be in a new line.  I missed this part when I reviewed it last time :(